app/views/shared/_expense_submission.html.erb

```diff
@@ -0,0 +1,102 @@
+<div class="container d-flex justify-content-center mt-5">
+  <h1>Submission Details</h1>
+</div>
+<div class="container mt-2 app-container">
+  <div class="row">
+    <div class="col-6">
+      <div class="row">
+        <div class="col-4">
+          <%= image_tag "site-logo.png" %>
+        </div>
+        <div class="col-8 d-flex align-items-center">
+          <h3>Expense Claim Form</h3>
+        </div>
+      </div>
+    </div>
+    <div class="col-6 d-flex align-items-center justify-content-end mt-5">
+      <span>BrainX technologies</span>
+    </div>
+  </div>
+
+  <div class="d-flex mt-5">
+    <div class="col-6">
+      <div class="d-flex">
+        <div class="col-4">
+          <span>Employee Name:</span>
+        </div>
+        <div class="col-8">
+          <span><%= @user.name %></span>
+        </div>
+      </div>
+    </div>
+    <div class="col-6">
+      <div class="row">
+        <div class="col-4">
+          <span>Job Title: </span>
+        </div>
+        <div class="col-8">
+          <span><%= @user.role.titleize %></span>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="row mt-4">
+    <div class="col-6">
+      <div class="row">
+        <div class="col-4">
+          <span>Employee ID :</span></div>
+        <div class="col-8 form-header">
+          <span><%= @user.emp_id %></span>
+        </div>
+      </div>
+    </div>
+    <div class="col-6">
+      <div class="row">
+        <div class="col-4">
+          <span>Department :</span>
+        </div>
+        <div class="col-8">
+          <span class="form-header"><%= @user.department %></span>
+        </div>
+      </div>
+    </div>
+  </div>
+
+  <div class="row mt-4">
+    <div class="col-6">
+      <span>Project Name :</span>
+      <span class="form-header"><%= @submission.data['project_name'] %></span>
+    </div>
+    <div class="col-6">
+    </div>
+  </div>
+
+  <table class="table table-bordered mt-4 form-table">
+    <thead>
+    <tr>
+      <% @form.data.keys.each do |key| %>
+        <% unless ["project_name"].include? key %>
```

Comment:
why not use if directly instead of array if we have only one element as project_name?